src/main/java/com/github/manolo8/darkbot/core/objects/facades/PlutusProxy.java

```diff
@@ -0,0 +1,18 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import com.github.manolo8.darkbot.core.utils.ByteUtils;
+
+import static com.github.manolo8.darkbot.Main.API;
+
+public class PlutusProxy extends Updatable implements eu.darkbot.api.API.Singleton {
```

Comment:
Would be nice if instead of just api.singleton, this implemented an actual interface on the api so that it's available in the plugin api